spec/models/case/base_spec.rb

```diff
@@ -79,45 +79,22 @@
     it { should validate_presence_of(:type)          }
   end
 
-  context 'an initial transition is created after any case is create' do 
-    let(:sar_case) { create :sar_case }
-    let(:offender_sar_case) { create :offender_sar_case }
-    let(:foi_case) { create :foi_case }
-    let(:ico_foi_case) { create :ico_foi_case }
-    let(:ico_sar_case) { create :ico_sar_case }
-    let(:overturned_ico_foi) { create :overturned_ico_foi }
-    let(:overturned_ico_sar) { create :overturned_ico_sar }
-
-    it 'sar_case' do 
-      expect(sar_case.transitions.size).to eq 1
-      expect(sar_case.transitions.first.event).to eq "create"
-    end
-
-    it 'offender_sar_case' do 
-      expect(offender_sar_case.transitions.size).to eq 1
-      expect(offender_sar_case.transitions.first.event).to eq "create"
-    end
-
-    it 'foi_case' do 
-      expect(foi_case.transitions.size).to eq 1
-      expect(foi_case.transitions.first.event).to eq "create"
-    end
-
-    it 'ico_foi_case' do 
-      expect(ico_foi_case.transitions.order('created_at').size).to eq 2
-      expect(ico_foi_case.transitions.first.event).to eq "create"
-      expect(ico_foi_case.transitions.second.event).not_to eq "create"
-    end
-
-    it 'overturned_ico_foi' do 
-      expect(overturned_ico_foi.transitions.size).to eq 1
-      expect(overturned_ico_foi.transitions.first.event).to eq "create"
+  describe 'a default transition is created after a case is created' do 
+    shared_examples 'case has default transition' do |case_type|
```

Comment:
I think this is loads better, what do you think?

Final wrinkle on this is, I think the shared examples (bizarrely and possibly unhelpfully) would normally go in their own folder `spec/support/shared_examples` - like a junk drawer. Almost all the ones in the app are in there

But I see Misha has also used them inline in one or two other places, so perfectly happy for this to live here 👍 

e.g. 
![image](https://user-images.githubusercontent.com/1161161/82358785-e8851480-99fe-11ea-9651-f68ba99c2467.png)

Reply:
thank you! was thinking to move it out as the above one did , but was thinking mine shared examples wont be used other places (at least for now), and plus I am not sure if I move it out, I may have other issues (trying to get less issue with tests right now, I got enough trouble with them already)